ci(benchmarks): Update formatter fixtures

leaysgur opened the pull request:
Fixes #15628 

- Remove unused `TestFiles::react()`
- Refactor private function to be inlined
- Add formatter fixtures
- Expand timeout 5 > 10s

leaysgur commented:
* **#15658** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15658?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15658?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.com/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=15658).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Dunqing commented:
I am fine with changing benchmark files, but adding more formatter benchmark files will cause CI to be a bit slow. What do you think? @Boshen

leaysgur commented:
I'll merge it for now.

After that, I'll compare the execution times with the original benchmark.
If it's clearly slower, I'll start deleting files that produce similar results.

leaysgur committed:
ci(benchmarks): Update formatter fixtures (#15658)

Fixes #15628

- Remove unused `TestFiles::react()`
- Refactor private function to be inlined
- Add formatter fixtures
- Expand timeout 5 > 10s

leaysgur commented:
NOTE: Since stopped downloading huge files, it actually feels a bit faster...
